bool success
string message
---
geometry_msgs/Pose2D pose
bool success
string message